server/routes.js

```diff
@@ -53,11 +53,62 @@ const createTransaction = (dbInstance, request, response) => {
         });
 };
 
+const getUsers = (dbInstance, request, response) => {
+    const { userId } = request.query;
+    db.get
+        .users(dbInstance, {userId})
+        .then(results => {
+            response.send(results.map(user => JSON.stringify(user, "", 2)));
+        })
+        .catch(err => {
+            response.send(`Error: ${err}`);
```

Comment:
One idea for API to simplify result processing: always send as JSON object with mandatory `success` (or `status`) field to display overall result type and `data` or `error` field with payload. Like 
`{success: true, data: <user>}` or `{success: false, error: "user not found"}`

Reply:
Fixed